osu.Game/Overlays/TabControlOverlayHeader.cs

```diff
@@ -108,17 +110,25 @@ private class OverlayHeaderTabItem : OverlayTabItem
                 public OverlayHeaderTabItem(T value)
                     : base(value)
                 {
-                    if (!(Value is Enum enumValue))
-                        Text.Text = Value.ToString().ToLower();
-                    else
+                    switch (Value)
                     {
-                        var localisableDescription = enumValue.GetLocalisableDescription();
-                        string nonLocalisableDescription = enumValue.GetDescription();
-
-                        // If localisable == non-localisable, then we must have a basic string, so .ToLower() is used.
-                        Text.Text = localisableDescription.Equals(nonLocalisableDescription)
-                            ? nonLocalisableDescription.ToLower()
-                            : localisableDescription;
+                        case LocalisableString localisableString:
+                            Text.Text = localisableString.ToLower();
+                            break;
+
+                        case Enum enumValue:
+                            var localisableDescription = enumValue.GetLocalisableDescription();
+                            string nonLocalisableDescription = enumValue.GetDescription();
+
+                            // If localisable == non-localisable, then we must have a basic string, so .ToLower() is used.
+                            Text.Text = localisableDescription.Equals(nonLocalisableDescription)
+                                ? nonLocalisableDescription.ToLower()
+                                : localisableDescription;
+                            break;
+
+                        default:
+                            Text.Text = Value.ToString().ToLower();
+                            break;
```

Comment:
Also, I would ideally expect all of this to be replaced just by this one line:
```csharp
Text.Text = Value.GetLocalisableDescription().ToLower();
```

But this will require `GetLocalisableDescription` to handle `LocalisableString` types (by returning themselves), and also confirm whether the `Enum` case here is due to `LocalisableString` not having `ToLower` supported at the point of writing it (I'm not really sure what it's doing there).

Reply:
@frenzibyte want to apply this change?

Reply:
Blamed through the `Enum` case and ended up all the way back to https://github.com/ppy/osu/pull/13516, so probably makes sense to always apply a lower case transform.